cli/lib/nf3.mjs

```diff
@@ -109,6 +109,8 @@ class Nf3 {
 
   clientAuthenticationKey;
 
+  NEXT_N_PROPOSERS = 3;
```

Comment:
This is the max number of Proposers we desire to send an off chain trx, right? If so, could we use a better name for this var?

Reply:
Possibly yes but it's code that I just refactored (moved) from elsewhere, so I'd rather change this as a different PR because I haven't taken time to deeply understand how this code segment works and I don't really want to add that complication here.